### ISSUE TITLE
[Fabric 1.19] music doesn't play 

### ISSUE THREAD

**Issue by JohnWeak1** [MC-U-Team/Music-Player#482]
### Bug description and reproduction steps

Music wont play and it errors occurs in the console each time I try to play

### Log files and Screenshots

[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Loading information for a YouTube track failed.
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:44)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:240)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.track(YoutubeAudioSourceManager.java:252)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.track(YoutubeAudioSourceManager.java:248)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.routeFromUrlWithVideoId(DefaultYoutubeLinkRouter.java:101)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.routeFromMainDomain(DefaultYoutubeLinkRouter.java:66)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:44)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:216)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:147)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:436)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:418)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:217)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.3.98-original
  os.arch: amd64
  os.name: Windows 10
  os.version: 10.0
  java.vendor: Oracle Corporation
  java.version: 17.0.4
  java.runtime.version: 17.0.4+11-LTS-179
  java.vm.version: 17.0.4+11-LTS-179
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: Caused by: java.lang.IllegalStateException: No match found
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at java.base/java.util.regex.Matcher.group(Matcher.java:644)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSignatureCipherManager.extractFromScript(YoutubeSignatureCipherManager.java:250)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSignatureCipherManager.getExtractedScript(YoutubeSignatureCipherManager.java:165)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.loadTrackInfoFromInnertube(DefaultYoutubeTrackDetailsLoader.java:207)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.load(DefaultYoutubeTrackDetailsLoader.java:56)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.loadDetails(DefaultYoutubeTrackDetailsLoader.java:44)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:228)
[20:15:56] [lava-daemon-pool-info-loader-2-thread-1/INFO]: [STDERR]: 	... 14 more

### Used minecraft version

1.19.x

### Used fabric version

fabric 0.14.9

### Used mod version

2.4.0.5

**Comment by HyCraftHD** [comment]
Update to 1.19.2 and this version: https://www.curseforge.com/minecraft/mc-mods/music-player/files/3989809
1.19.0 is out of support.

**Comment by KajeArch** [comment]
> Update to 1.19.2 and this version: https://www.curseforge.com/minecraft/mc-mods/music-player/files/3989809 1.19.0 is out of support.

I'd argue that MC 1.19 is still being one of the most used versions right now, so I'm not quite sure why support has been dropped for this version. So, what exactly was changed to make it work again? If anything I'd like to see a quick hotfix for this issue or a way to do it myself. I play on servers that still use 1.19 and due to the problematic nature of the chat reporting system, there seems to be a hesitance of server owners to update. @HyCraftHD 

Anyway, I posted a similar issue just right now before seeing this one. My first suspicion was that YouTube has changed something in their API, hence I'm not sure if the newer version actually fixed this issue. #483
